jenkins-agent.ps1

```diff
@@ -67,6 +69,10 @@ if(![System.String]::IsNullOrWhiteSpace($Cmd)) {
 		$WorkDir = " -workDir `"$WorkDir`""
 	}
 
+	if("true".Equals($Env:JENKINS_WEB_SOCKET)) {
```

Comment:
Tested a similar condition in a local PowerShell prompt, but did not try this script in context.

Reply:
You don't need to use .Equals in .NET like you do in Java. You can just do `$env:JENKINS_WEB_SOCKET -eq "true"`

This is more Powershell-ish

Reply:
And seems to work with a null value too. Will edit accordingly.